inference-engine/tests/functional/inference_engine/ir_serialization/rt_info_deserialization.cpp

```diff
@@ -565,6 +565,147 @@ TEST_F(RTInfoDeserialization, NodeV11) {
     }
 }
 
+
+TEST_F(RTInfoDeserialization, NodeV11_uint8) {
+    std::string model = R"V0G0N(
+<net name="Network" version="11">
+    <layers>
+        <layer name="in1" type="Parameter" id="0" version="opset8">
+            <data element_type="u8" shape="1,22,22,3"/>
+            <rt_info>
+                <attribute name="old_api_map_element_type" version="0" value="f16"/>
+                <attribute name="old_api_map_order" version="0" value="0,2,3,1"/>
+                <attribute name="fused_names" version="0" value="in1"/>
+            </rt_info>
+            <output>
+                <port id="0" precision="FP32" names="input_tensor">
+                    <dim>1</dim>
+                    <dim>3</dim>
+                    <dim>22</dim>
+                    <dim>22</dim>
```

Comment:
Yes, shapes are not aligned here. I think I need to remove old_api_map_order from this test IR, as only old_api_map_element_type is tested.

Reply:
I corrected shapes in this test. 
Transpose is not necessarily stands after Parameter. This happens if next operations require NCHW then MO inserts transposes. In case if there are no such operation MO automatically disables inner NHWC->NCHW layout change.